pkgs/by-name/tf/tftui/package.nix

```
@@ -19,15 +19,17 @@ python3.pkgs.buildPythonApplication rec {
   };
 
   pythonRelaxDeps = [
+    "posthog"
     "textual"
   ];
 
   nativeBuildInputs = with python3.pkgs; [
```

```suggestion
  nativeBuildInputs = with python3Packages; [
```
Still, migrate to `python3Packages` please ;)

---

$BADWORD. Sorry.

---

Done